docs/streams/developer-guide/dsl-api.html

```diff
@@ -1050,8 +1050,8 @@ <h4 class="anchor-heading"><a id="streams_concepts_globalktable" class="anchor-l
                                         more than once for a key as a result of having received input tombstone records for that key (see below).</li>
                                     <li>When the first non-<code class="docutils literal"><span class="pre">null</span></code> value is received for a key (e.g.,  INSERT), then only the adder is called.</li>
                                     <li>When subsequent non-<code class="docutils literal"><span class="pre">null</span></code> values are received for a key (e.g.,  UPDATE), then (1) the subtractor is
-                                        called with the old value as stored in the table and (2) the adder is called with the new value of the
-                                        input record that was just received.  The order of execution for the subtractor and adder is not defined.</li>
+                                        first called with the old value as stored in the table and then (2) the adder is called with the new value of the
+                                        input record that was just received.</li>
```

Comment:
Well, for the regular case in which the key changes, the order is still not defined...

We would need to be more precise and say:
> If and only if the extracted grouping key of the old and new value is the same, the subtractor will be called before the adder. The detection of this case depends on the correct implementation of the <code>equals()</code> method of the extracted key type.